src/en/tut-lxd.md

```diff
@@ -140,10 +147,10 @@ The `juju bootstrap` command is used to create the controller. The command
 expects a name (for referencing this controller) and a cloud to use. The LXD
 'cloud' is known as 'localhost' to Juju.
 
-For our localhost cloud, we will create a controller called 'lxd-test':
+For our localhost cloud, we will create a controller called 'lxd-controller':
 
 ```bash
-juju bootstrap localhost lxd-test
+juju bootstrap lxd lxd-controller
```

Comment:
why change localhost to lxd?

also, I think we may have to look at some of these names. a lot of work has gone into trying to make the output tables more concise, so having 15 character names for things isn't helpful

Reply:
I can change it back to 'localhost' if you want, but I went with the majority (which includes the related `clouds-LXD.md` file):

**lxd**
```
charms-deploying.md:juju bootstrap --constraints="mem=4G" lxd lxd-controller
clouds-LXD.md:juju bootstrap lxd lxd-test
controllers-config.md:juju bootstrap --config allow-model-access=true lxd
controllers-creating.md:juju bootstrap lxd lxd-xenial
controllers-creating.md:juju bootstrap --bootstrap-series=trusty lxd lxd-trusty
controllers-creating.md:juju bootstrap --constraints="mem=4G" lxd lxd-xenial
controllers-creating.md:juju bootstrap --constraints="mem=4G" lxd
controllers-creating.md:juju bootstrap --config bootstrap-timeout=900 lxd lxd-faraway
models-config.md:juju bootstrap --config image-stream=daily lxd lxd-daily
users-workflows.md:juju bootstrap lxd lxd-controller-1
```

**localhost**
```
controllers-creating.md:juju bootstrap localhost lxd-new --no-switch
troubleshooting.md:juju bootstrap localhost --debug
tut-lxd.md:juju bootstrap localhost lxd-test
```

Reply:
either works but I believe 'lxd' was the 'old' naming scheme. I would prefer to go with localhost as that's how it is now labelled in the status output.

Reply:
OK